transport/http/transport_test.go

```diff
@@ -297,3 +297,11 @@ func TestTransportClientWithMaxIdleConnections(t *testing.T) {
 
 	assert.NotNil(t, transport.client)
 }
+
+func TestTransportClientWithSocks5Proxy(t *testing.T) {
+	// Unfortunately the client.transport.Dialer is obfuscated in the client, so we can't really
```

Comment:
I'm open to ideas on how to unit test this? We're wiring up a thing from x/net/proxy to net/http - this code is there because it follows the pattern of the previous 2 unit tests in that file - saying essentially `we can't really test this because we're plugging two things together`.